test/tools/unified-spec-runner/operations.ts

```diff
@@ -206,17 +206,16 @@ operations.set('createChangeStream', async ({ entities, operation }) => {
     resumeAfter: operation.arguments.resumeAfter,
     startAfter: operation.arguments.startAfter,
     startAtOperationTime: operation.arguments.startAtOperationTime,
-    batchSize: operation.arguments.batchSize
+    batchSize: operation.arguments.batchSize,
+    comment: operation.arguments.comment
   });
-  changeStream.eventCollector = new EventCollector(changeStream, ['init', 'change', 'error']);
 
-  return new Promise((resolve, reject) => {
-    const timeout = setTimeout(() => {
-      reject(new Error('Change stream never started'));
-    }, 2000);
+  changeStream.eventCollector = new EventCollector(changeStream, ['init', 'error']);
```

Comment:
I'm going to respond here about this comment as well as for your question about why I skipped the `'Test consecutive resume'` test.

Our unified spec runner was not spec compliant for change streams.  The `createChangeStream` operation is only supposed to perform the initial `aggregate` call.  Our stream-based implementation of change streams is lazy - we don't perform any calls until a `change` listener is added and once one is, we immediately perform the aggregate call and _and_ start sending getMores.  One solution would be to refactor our change streams implementation to perform the `aggregate` call on creation, and `getMores` when a `change` listener is added but that would be a decent refactor.  Instead, our unified tests now use the cursor-based implementation.

`'Test consecutive resume'` now fails because our iterator-based change stream implementation doesn't resume properly (see https://jira.mongodb.org/browse/NODE-4126).  I'll update the jira ticket to reflect this.

I think we're safe to actually remove the event collector entirely though.